README.md

```
@@ -45,11 +115,11 @@ Description: Implementation of OpenID for Verifiable Presentations - in Kotlin
    > Note: If the Authorization request includes an `mso_mdoc` format VP, it can only use the `direct_post.jwt` response mode, as required by the ISO-18013-7 specification. Other supported response mode (`direct_post`) is not applicable.
 3. `iar-post` :
     - Authorization Response is constructed in unencrypted format.
-    - Sample Authorization response structure:
+    - Sample Authorization response structure of DIF Presentation Submission format:
    ```shell
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)